Simplified/NYPLKeychainManager.swift

```diff
@@ -35,6 +36,7 @@ import Foundation
     #endif
     updateKeychainForBackgroundFetch()
     manageFeedbooksData()
+    manageFeedbookDrmPrivateKey()
```

Comment:
just a note for the future: this code is run during `application:didFinishLaunchingWithOptions:` which has to finish as quickly as possible, so we should start thinking at different ways to do this kind of initialization instead of tagging it inside that method. This is manipulating the keychain, so I am just wondering if there's something to worry about here.

Reply:
I understand, inside the code checks vendor's key date (date is stored in UserDefaults) and if no date or date < today, creates URLSession.shared.dataTask, which are running in a separate queue, so this functions quits quickly. No keychain manipulation at that point.